typescript/sdk/src/aws/validator.ts

```diff
@@ -39,7 +39,7 @@ export class S3Validator extends BaseValidator {
         const s3Config = {
           bucket: pieces[0],
           region: pieces[1],
-          folder: pieces[2],
+          folder: pieces.slice(2).join('/'),
```

Comment:
might be my smooth brain but can you give me an example of before/after this fix pls

Reply:
The existing implementation: Coinsummer are storing the objects in a sub directory, `s3://cs-avs-hyperlane/us-east-1/validator/avs1/ethereum`, we are looking in `validator` and not finding anything.

new implementation will look in `validator/avs1/ethereum`